packages/dd-trace/src/config.js

```diff
@@ -566,6 +566,9 @@ class Config {
     this._setValue(defaults, 'url', undefined)
     this._setValue(defaults, 'version', pkg.version)
     this._setValue(defaults, 'instrumentation_config_id', undefined)
+    this._setValue(defaults, 'baggagePropagation', true)
+    this._setValue(defaults, 'baggageInject', true)
+    this._setValue(defaults, 'baggageExtract', true)
```

Comment:
The idea was that the separate `inject`/`extract` settings take precedence if they are explicitly set by the user. I'm not sure how to handle this in Node.js, but you may want to set the default for those two as `undefined`.

This is similar to what we do with
```
DD_TRACE_PROPAGATION_STYLE (use for both extract and inject if they are not set)
DD_TRACE_PROPAGATION_STYLE_EXTRACT (overrides DD_TRACE_PROPAGATION_STYLE for extraction only)
DD_TRACE_PROPAGATION_STYLE_INJECT (overrides DD_TRACE_PROPAGATION_STYLE for injection only)
```